### DISCUSSION TITLE
Photo & video editing capability what we get with simple gallery pro

### DISCUSSION THREAD

**Discussion by soumyadip69** [FossifyOrg/General-Discussion#382] [Ideas]
Is it possible to add editing features just like what we have with simple gallery pro ? BTW really thanks u for ur contribution to this program

**Comment by Aga-C** [comment]
It surely needs some time, because we can't use the old editor due to its license. Maybe we will work on a more advanced editor (https://github.com/orgs/FossifyOrg/discussions/9#discussioncomment-7790824).

**Reply by soumyadip69** [reply]
Ah i see , take ur time as ur comfort,thanks for the info 😊

**Reply by e-t-l** [reply]
I came here to ask about the same thing. I currently use Simple Gallery Pro. I intend to switch all my SMT apps to their Fossify versions, but if I change from Simple Gallery to Fossify Gallery right now, I will be losing functionality, right?

I don't know how SDKs and licensing works, so maybe this won't be feasible, but is there any way to get the bug fixes and other good updates that you're adding in the latest versions of Fossify Gallery *without* also losing the functionality of SG Pro?

**Reply by Aga-C** [reply]
@e-t-l You can always use both apps at once. Fossify Gallery for browsing and viewing your photos/videos and Simple Gallery Pro just for editing (while clicking edit you're anyway prompted what app you want to use). I use it like this.
However, you have to be aware to not update the app if you have it from Google Play.

**Reply by naveensingh** [reply]
You can also download SG with the proprietary editor from https://github.com/SimpleMobileTools/Simple-Gallery/releases/download/6.28.1/gallery-396-proprietary-release.apk

I don't know if it has the same signature as the Google Play version but if it doesn't, automatic updates won't happen.

**Comment by bjlockie** [comment]
I a Fossify Gallery Basic Editor but not the SG Advanced Editor.
I think I've only used Crop.
What does the Advanced Editor do?

**Reply by Aga-C** [reply]
It had more editing capabilities like stickers, overlays, advanced drawing, etc.

**Comment by naveensingh** [comment]
Closing as there's already an issue for basic video editing and the photo editor will be improved over time.
